docs/chapter-2.md

```diff
@@ -9,53 +9,79 @@ sidebar_position: 2
 
 If you already have NodeJS and Angular CLI on your machine, skip this step. Otherwise, do the following:
 
-1. Download NodeJS
-2. Install it on your machine. This may take several minutes
-3. Install Angular CLI: open your command line tool/terminal and run the following command: `npm install @angular/cli -g`
-4. We are done. You can read more about Angular CLI in the official docs
+1. Download and install [NodeJS](https://nodejs.org)
+1. Install the [Angular CLI](https://angular.io/cli) by running the following command:  
+
+  ```sh
+  npm install -g @angular/cli
+  ```
 
 ### Creating the project
 
-As I mentioned in the previous chapter, we are going to create an Angular finance logging application with NgRx. Let's create this project and get started:
+As I mentioned in the previous chapter, we are going to create an Angular finance logging application with NgRx.
 
-1. Open a preferred directory of yours
-2. Run the following command `ng new finance-logger`
-3. You can select a number of options, add routing, SCSS, and so on. Those options are irrelevant to this tutorial
-4. Wait a bit
-5. Application has been created
+Let's create this project and get started by creating a new Angular project:
 
-Now let's add NgRx to the party:
+```sh
+ng new finance-logger --minimal --defaults
```

Comment:
Doesn't `minimal` create a project without unit tests? I want to have chapter about unit testing NgRx in the future, so if this is the case, let's find another approach (and explain why we run the command in that particular way)

Reply:
I get it! I didn't see any chapter on that so I was thinking about keeping the demo as minimal as possible so that users can focus on the described parts but if this is part of the course I'll remove that argument

Reply:
Thanks!